autogpt_platform/backend/backend/integrations/oauth/google.py

```diff
@@ -9,14 +12,21 @@
 
 from .base import BaseOAuthHandler
 
+logger = logging.getLogger(__name__)
+
 
 class GoogleOAuthHandler(BaseOAuthHandler):
     """
     Based on the documentation at https://developers.google.com/identity/protocols/oauth2/web-server
-    """  # noqa
+    """
 
     PROVIDER_NAME = "google"
     EMAIL_ENDPOINT = "https://www.googleapis.com/oauth2/v2/userinfo"
+    DEFAULT_SCOPES = [
+        "https://www.googleapis.com/auth/userinfo.email",
+        "https://www.googleapis.com/auth/userinfo.profile",
+        "openid",
+    ]
```

Comment:
All google blocks MUST use these scopes so lets add them here so we don't lose them

Reply:
google will helpfully add them for you which breaks validation if you don't have them btw